scripts/utils/yarn.ts

```diff
@@ -36,7 +35,6 @@ export const addPackageResolutions = async ({ cwd, dryRun }: YarnOptions) => {
   const packageJson = JSON.parse(content);
   packageJson.resolutions = {
     ...packageJson.resolutions,
-    ...storybookVersions,
```

Comment:
This change should be explained.
Why was it needed? Why is it no longer needed?